src/lib.rs

```diff
@@ -2053,6 +2053,16 @@ mod test {
         assert_eq!(result.to_string(), "<em>This is</em><p>still here!</p>");
     }
     #[test]
+    fn clean_removed_default_tag() {
+        let fragment = "<em>This is</em><script><a>Hello!</a></script><p>still here!</p>";
+        let result = String::from(Builder::new()
+            .rm_tags(std::iter::once("a"))
+            .rm_tag_attributes("a", ::std::iter::once("href").chain(::std::iter::once("hreflang")))
```

Comment:
This is obviously not the right way to do this, but I couldn't figure out any other way.

Related question: could these methods take 
```rust
pub fn add_tags<I>(&mut self, it: I) -> &mut Self
where
    I: IntoIterator,
    I::Item: AsRef<str>
```

which would let you call them like this:

```rust
.add_tags(&["a", "em", "script"])
```

Reply:
I pushed a commit with more reasonable code. And filed #92